README.md

```diff
@@ -412,7 +417,7 @@ $ kubectl delete -f deploy/service_account.yaml
 
 ## Reference implementation
 
-The above walkthrough follows the actual implementation process used to produce the `memcached-operator` in the SDK [samples repo][repo_sdk_samples_memcached].
+Also, please feel free to check an `memcached-operator` project in the SDK [samples repo][repo_sdk_samples_memcached].
```

Comment:
Whats the motivation for this change? IMO the original was clearer.

Reply:
Yeah all the inter-repo links and dependencies make this a little problematic since things can change in one place and not in others.

How about this?
```suggestion
The above walkthrough follows a similar implementation process to the one used to produce the `memcached-operator` in the SDK [samples repo][repo_sdk_samples_memcached].
```